centralserver/i18n/management/commands/update_language_packs.py

```diff
@@ -558,15 +558,9 @@ def produce_outputs(src_po_files, dest_path, lang_code):
                 js_po_file.save_as_mofile(js_mo_file)
             else:
                 # Make sure we only concatenate .po files of the same version that we need.
-                versioned_po_filename = os.path.join("versioned", "%s-django") % (version,)
-                kalite_po_filename = os.path.join("KA Lite UI", "kalite-%s.po") % (lang_code,)
-                if versioned_po_filename in src_file or kalite_po_filename in src_file:
-                    logging.debug('Concatenating %s with %s...' % (src_file, build_file))
-                    src_po = polib.pofile(src_file)
-                    build_po.merge(src_po)
-                else:
-                    logging.debug("Ignoring %s because it's NOT for version %s." %
-                                  (src_file, version,))
+                src_po = polib.pofile(src_file)
```

Comment:
Please don't just remove the `if` statement here because we use it to build versioned .mo files as per #210.

There must be a way to preserve that functionality while fixing this issue.